docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,41 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Input
+
+<p class="description">The `Input` component allows users to enter and edit text.</p>
+
+## Basic input
+
+```js
+import InputUnstyled from '@mui/base/InputUnstyled';
+```
+
+{{"demo": "UnstyledInputBasic.js", "defaultCodeOpen": false}}
+
+## Adornments
+
+Sometimes it is useful to add some kind of prefix, a suffix, or an action to an input.
+You can achieve this by using the adornment props.
```

Comment:
Tightening this up. Also, should "Adornment" get back ticks here?

```suggestion
You can use the adornment props to add a prefix, suffix, or an action to an input.
```

Reply:
I've applied https://github.com/mui/material-ui/pull/31881/commits/cb792c12257634f011b858794a2e4112b2aa8057 combining your two comments.